types/decentlab/IAM/lifecycle.schema.json

```diff
@@ -23,7 +23,8 @@
     "protocolVersion": {
       "title": "Protocol Version",
       "description": "Protocol Version",
-      "type": "integer"
+      "type": "integer",
+      "hideFromKpis": false
```

Comment:
should be true ;)

Reply:
lol, notifications go to my personal mail, just saw this